app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+end
```

Comment:
- [ ] Kindly add a method that updates the likes counter for a post.

### Requirement:
![image](https://github.com/mahdinoori2000/blog-app/assets/50990925/65396ba9-030b-4638-a07d-c0a5a2310f57)


- [Optional] It will be good if you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new comment has been saved to database, then make the `update_likes_counter method` private